### DISCUSSION TITLE
Support for skipping objects in sequence

### DISCUSSION THREAD

**Discussion by dhysuiej** [mriscoc/Ender3V2S1#1088] [Q&A] [answered]
I'm using SuperSlicer right now and there's an option in the slicer printer settings to add some custom g-code between objects in a sequence. I'm wondering if this firmware has an option to skip the current object being printed, in case there are any issues, and start the next object in the sequence, instead of having to cancel the whole print and start over.

**Comment by mriscoc** [comment, marked as answer]
M486 for cancel object is enabled in this firmware, you can check that in the configuration files:
```h
/**
 * Cancel Objects
 *
 * Implement M486 to allow Marlin to skip objects
 */
#define CANCEL_OBJECTS  // MRiscoC Enabled M486 to skip objects
#if ENABLED(CANCEL_OBJECTS)
  #define CANCEL_OBJECTS_REPORTING // Emit the current object as a status message
#endif
```
I use a cancel object plugin for Octoprint.

**Reply by dhysuiej** [reply]
Thanks. So, if I understand correctly I would need to send M486 to the printer from an app that supports sending custom commands. ¿Are there any plans to support activating this command directly from the display on the printer?
